pkg/sysregistries/system_registries_test.go

```diff
@@ -0,0 +1,68 @@
+package sysregistries
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+var testConfig = []byte("")
+
+func init() {
+	readConf = func() ([]byte, error) {
+		return testConfig, nil
+	}
+}
+
+func TestGetRegistriesWithBlankData(t *testing.T) {
+	answer := []string(nil)
+	testConfig = []byte("")
+	registriesConfig, err := GetRegistries()
+	if err != nil {
```

Comment:
Since you've already pulled in the package: assert.nil(t, err) would result in less code.